staging/src/k8s.io/component-helpers/storage/volume/helpers.go

```diff
@@ -16,7 +16,13 @@ limitations under the License.
 
 package volume
 
-import v1 "k8s.io/api/core/v1"
+import (
```

Comment:
There doesn't seem to be a reason to have 2 files. All the functions refer to pv/pvc.

Maybe just merge them in a separate commit (or PR). The way you have the PR now helps a lot to see that nothing really changed and it's just a move.

Reply:
Sure I will do it another PR as a follow up.

Reply:
Don't forget about this please.